eZ/Bundle/EzPublishCoreBundle/Command/ReindexCommand.php

```diff
@@ -233,12 +244,16 @@ protected function indexIncrementally(InputInterface $input, OutputInterface $ou
         } else {
             // if we only have one process, or less iterations to warrant running several, we index it all inline
             foreach ($this->fetchIteration($stmt, $iterationCount) as $contentIds) {
-                $this->searchIndexer->updateSearchIndex($contentIds, $commit);
+                $unindexableContentIds += $this->searchIndexer->updateSearchIndex($contentIds, $commit, $continueOnError);
                 $progress->advance(1);
             }
         }
 
         $progress->finish();
+
+        if ($unindexableContentIds) {
+            $this->printUnindexableContentIds($output, $unindexableContentIds);
```

Comment:
missing return for error code, also need to decide in what conditions parallel processes indexing should fail or not, as it afaik will otherwise silently index all and not make user aware there where errors.

Reply:
Yes, you are absolutely right. I'm going to rewrite this part.